### PR TITLE
`Paywalls Tester`: group live paywalls by template

### PR DESCRIPTION
**Pull request by NachoSoto**
image

### CODE REVIEW

**Comment by NachoSoto** [comment]
I recommend reviewing this ignoring whitespaces.

File: Tests/TestingApps/PaywallsTester/PaywallsTester/Views/OfferingsList.swift [hunk 1]
```diff
@@ -27,11 +36,21 @@ struct OfferingsList: View {
         }
         .task {
             do {
+                let offerings = try await Purchases.shared.offerings()
+                    .all
+                    .map(\.value)
+                    .sorted { $0.serverDescription > $1.serverDescription }
```
Comment (aboedo) [inline Tests/TestingApps/PaywallsTester/PaywallsTester/Views/OfferingsList.swift:42]: we can delete this sorting now, right? we have it again on line 51

**Review by aboedo — APPROVED** [review]

File: Tests/TestingApps/PaywallsTester/PaywallsTester/Views/OfferingsList.swift (hunk 1 above)
Comment (NachoSoto) [reply Tests/TestingApps/PaywallsTester/PaywallsTester/Views/OfferingsList.swift:42]: That one is sorting keys (templates). This is sorting the values (offerings) by name.

Comment (aboedo) [reply Tests/TestingApps/PaywallsTester/PaywallsTester/Views/OfferingsList.swift:42]: Ohh, sorry, makes sense.

**Pull request merged**